Initial version

tapdancemobile opened the pull request:
Take a look at this - thanks!

tapdancemobile committed:
added react-native .gitignore and updated readme

tapdancemobile committed:
added essential files - cleaning up a bit

tapdancemobile committed:
added essential files - cleaning up a bit

tapdancemobile committed:
added package-lock.json

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/ethereumproject/emerald-mobile-wallet?pullRequest=1) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/ethereumproject/emerald-mobile-wallet?pullRequest=1) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/ethereumproject/emerald-mobile-wallet?pullRequest=1) it.</sub>

shanejonas commented on index.android.js:
```diff
@@ -0,0 +1,65 @@
+/**
```
we should probably just have one entry file

tapdancemobile replied:
Got it... That's fine. I think that's just RN convention

shanejonas replied:
Lets just make one entry file that should work for both: `index.js`

shanejonas commented on ios/emerald_mobile_wallet-tvOS/Info.plist:
```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
is this tvOS file necessary?

tapdancemobile replied:
No that's just boilerplate XCode stuff

shanejonas commented on ios/emerald_mobile_wallet-tvOSTests/Info.plist:
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
is this tvOS file necessary?

shanejonas commented on ios/emerald_mobile_wallet.xcodeproj/xcshareddata/xcschemes/emerald_mobile_wallet-tvOS.xcscheme:
```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
more tvOS

shanejonas replied:
remove tvOS file

shanejonas replied:
remove tvOS file

shanejonas commented on rust/emerald_mobile_wallet/src/lib.rs:
```diff
@@ -0,0 +1,19 @@
+#[cfg(feature = "jni")]
```
dont think we need this?

shanejonas commented on ios/emerald_mobile_wallet/AppDelegate.h:
```diff
@@ -0,0 +1,16 @@
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
```
BSD

shanejonas commented on ios/emerald_mobile_wallet/AppDelegate.m:
```diff
@@ -0,0 +1,37 @@
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
```
BSD

tapdancemobile replied:
All iOS apps use this... Not sure we can get away with not having it. Generated by the platform but I can rewrite with the same content?

shanejonas commented on ios/emerald_mobile_wallet/main.m:
```diff
@@ -0,0 +1,18 @@
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
```
BSD

tapdancemobile replied:
All iOS apps use this... Not sure we can get away with not having it

tapdancemobile replied:
This is generated by the React-Native platform

tapdancemobile replied:
React-Native is not under MIT license, though they just put React under MIT

shanejonas commented on ios/emerald_mobile_walletTests/emerald_mobile_walletTests.m:
```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
```
maybe remove these tests. i dont think we will need them

tapdancemobile replied:
I can remove the ios tests... you're right - we'd probably use jasmine or jest

tapdancemobile committed:
- removed tvOS target and tests
- removed main target tests
- removed BSD license to defer to root Apache license
- removed unneeded jni statement

tapdancemobile committed:
- removed tvOS scheme
- using single index.js instead of platform dependent

tapdancemobile commented:
@shanejonas - done... thanks!
